vue/components/index.js

```diff
@@ -13,6 +13,7 @@ import { ProgressBarPlatforme } from "./ui/progress-bar-platforme/progress-bar-p
 import { SearchPlatforme } from "./ui/search-platforme/search-platforme.vue";
 import { SidePlatforme } from "./ui/side-platforme/side-platforme.vue";
 import { TablePlatforme } from "./ui/table-platforme/table-platforme.vue";
+import { LoggingContainer } from "./ui/logging-container-platforme/logging-container-platforme.vue";

```

Comment:
Let's rename to `LogPlatforme` for coherence with the stuff that already exists.